log10/load.py

```diff
@@ -692,6 +780,11 @@ def log10(module, DEBUG_=False, USE_ASYNC_=True):
         attr = module.resources.completions.Completions
```

Comment:
remove completions.create, https://docs.anthropic.com/claude/reference/migrating-from-text-completions-to-messages